R/create_individual_file.R

```diff
@@ -90,14 +92,17 @@ create_individual_file <- function(
     recode_gender() %>%
     clean_individual_file(year) %>%
     join_cohort_lookups(year) %>%
+    write_temp_data(data, year, file_name = "indiv_temp2", test_mode) %>%
```

Comment:
i think we could move this up to line 84 and line 89 so that we save a temp file right after the aggregate? i remember that function takes a while to process